src/components/utils.js

```diff
@@ -44,7 +44,12 @@ export function cellLayerDefaultProps(cells, updateStatus, setCellHighlight, set
   };
 }
 
-export const DEFAULT_COLOR = [128, 128, 128];
+export const DEFAULT_DARK_COLOR = [50, 50, 50];
+export const DEFAULT_LIGHT_COLOR = [200, 200, 200];
+
+export function getDefaultColor(theme) {
+  return theme === 'dark' ? DEFAULT_DARK_COLOR : DEFAULT_LIGHT_COLOR;
+}
```

Comment:
I think it could be worth revisiting how themes are handled.  Had to change a lot of code to get this to work - feels like it almost should be present on `window` but I think that is out of scope here.

Reply:
True, or maybe allowing `null` values in more places, and converting to colors closer to rendering time, but that also may be tricky.